showmethekey-cli/meson.build

```diff
@@ -19,5 +19,5 @@ executable(
   dependencies: cli_dependencies,
   include_directories: cli_include_directories,
   install: true,
-  install_dir: get_option('prefix') / 'bin'
+  install_dir: get_option('prefix') / get_option('bindir')
```

Comment:
The default install_dir for an executable is already `install_dir: get_option('bindir')`, and you do not need to join the prefix to an install_dir path since those are always relative to the prefix already (unless of course they get specified as an absolute path).

The typical reason to join prefix and bindir together is when writing out the string to a file -- such as the desktop file getting configured above.

Reply:
Thanks, I didn’t know that. Fixed in ff967cd4832b0685260c1a66ad9a3442a8cb7b26.